src/cli/manage.rs

```diff
@@ -44,6 +44,13 @@ pub struct Manage {
     /// The configuration source for a management server.
     #[clap(subcommand)]
     pub provider: crate::config::Providers,
+    /// If specified, filters the available gameserver addresses to the one that
+    /// matches the specified type
```

Comment:
Just noting that this will update the docs as well, since they are autogenerated - so no need for changes there.